src/basics.php

```diff
@@ -142,7 +142,13 @@ function dd($var, $showHtml = null): void
             'line' => $trace[0]['line'],
             'file' => $trace[0]['file'],
         ];
-
+        
+        $origin = substr($_SERVER['HTTP_REFERER'], 0, (strlen($_SERVER['HTTP_REFERER']) - 1));
+        $headers = ['authorization', 'content-type', 'api-token'];
+        header('Access-Control-Allow-Credentials: true');
+        header('Access-Control-Allow-Headers: '.implode($headers, ' '));
+        header('Access-Control-Allow-Origin:'.str_replace('\/', '', $origin));
```

Comment:
I dont think header output are a good idea inside such a generic helper function, especially since this is also used in CLI context.

Reply:
make sense, need to verify SAPI env.